build/manifest/main.go

```diff
@@ -10,19 +10,36 @@ import (
 	"github.com/pkg/errors"
 )
 
-const pluginIdGoFileTemplate = `package main
-
-var manifest = struct {
-	Id      string
-	Version string
-}{
-	Id:      "%s",
-	Version: "%s",
+const pluginIDGoFileTemplate = `// This file is automatically generated. Do not modify it manually.
+
+package main
```

Comment:
@hanzei is was there some funky merging here?  package main defined more than once..

Reply:
I'm not sure I'm getting was the issue is. The generated file gets placed into `server`, which contains all needed `.go` files for the plugin to build. They are all in `package main`.

Reply:
I was thrown off by the additional `package main` line while skimming through the files and didn't see that is included as a string.